crates/cheatnet/tests/cheatcodes/prank.rs

```diff
@@ -300,7 +300,7 @@ fn prank_cairo0_callback() {
         ExtendedStateReader {
             dict_state_reader: build_testing_state(),
             fork_state_reader: Some(ForkStateReader::new(
-                "http://188.34.188.184:6060/rpc/v0_6".parse().unwrap(),
```

Comment:
Why don't we have a constant for this yet 👁️👄👁️

Reply:
We have multiple ones even xD We can prob clean it up but not in this PR